scripts/module/keybinder.nut

```diff
@@ -3,26 +3,26 @@ function newBindKey( keycode ) {
 }
 
 keys <- {
-  "backspace" : newBindKey(0x08),
-  "return" : newBindKey(0x0D),
-  "lshift" : newBindKey(0xA0),
-  "rshift" : newBindKey(0xA1),
-  "lctrl" : newBindKey(0xA2),
-  "rctrl" : newBindKey(0xA3)
+  "backspace" : 0x08,
+  "return" : 0x0D,
+  "lshift" : 0xA0,
+  "rshift" : 0xA1,
+  "lctrl" : 0xA2,
+  "rctrl" : 0xA3
 };
 
 function onScriptLoad() {
   binds <- array(100, null);
```

Comment:
I still want to store binds server side too as I imagine it could be nice to store them in a DB/.ini and send them back `onPlayerJoin`.

Reply:
Done.